plugin-locationcomponent/src/test/java/com/mapbox/maps/plugin/locationcomponent/generated/LocationComponentAttributeParserTest.kt

```diff
@@ -123,6 +124,12 @@ class LocationComponentAttributeParserTest {
     val settings = LocationComponentAttributeParser.parseLocationComponentSettings(context, attrs, 1.2f)
     assertEquals(null, settings.layerBelow)
   }
+  @Test
+  fun puckBearingSourceTest() {
+    every { typedArray.getInt(any(), any()) } returns PuckBearingSource.HEADING.ordinal
```

Comment:
a bit curious why don't we have another test for another enum value generated

Reply:
Since this is generated and would affect other parts, I will create a new ticket for this.